README.md

```diff
@@ -126,6 +133,10 @@ Requirements:
 - A DID string MUST be provided.
 - The provided DID MUST be compliant with the `did:itn` method.
 
+Here is how resolve operation works:
+
+- The ITN Agent validates the provided DID, retrieves the DID Document from the federated content-addressable storage solution, retrieves the DID Document proof from one or more utilized ledger (either private or public) and returns the DID Document along with the DID Document proof to the requestor.
```

Comment:
ITN Agent is not involved in resolution process. The request goes directly to the DID Resolver.

Reply:
Updated.